.circleci/generate_config.sh

```diff
@@ -2,11 +2,11 @@
 
 # TODO: This is flaky as it relies on coarse directory-level diffs.
 git diff --quiet HEAD $(git merge-base main HEAD) -- compiler/ prelude/ skfs/
-skc=$?
+skc=1
 git diff --quiet HEAD $(git merge-base main HEAD) -- skfs/ compiler/runtime/
-skfs=$?
+skfs=1
 git diff --quiet HEAD $(git merge-base main HEAD) -- sql/ sqlparser/
-skdb=$?
+skdb=1
```

Comment:
Did you mean to leave this in?

Reply:
Nope, this is part of the `[DoNotCommit] Force CI.` commit.